observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="distr-tracing-jaeger-configuring"]
-= Configuring and deploying the distributed tracing platform Jaeger
+= Configuring
```

Comment:
I would be inclined to keep this as is since it would be good for user to stay informed about the page they have landed in. Corresponding changes in topic maps file look good to me since it keeps the panel neat however, the page headings should be descriptive enough for better user experience as well as from SEO standpoint.
The same comment applies to all the modified headings.